packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -273,7 +280,27 @@ export class CXLMarketingNavElement extends LitElement {
         );
 
         // Populate.
-        contextMenu.items = this._createContextMenuItems(menuItem.children);
+        const contextMenuItems = this._createContextMenuItems(menuItem.children);
+        contextMenu.addEventListener('opened-changed', () => {
+          contextMenu.items = contextMenuItems;
+          const listBox = document.querySelector('vaadin-context-menu-list-box');
```

Comment:
This is guaranteed to be unique? Any available opportunities to become more specific with the selector?

Reply:
The top level menu will always be first in the document, I do not know a way to add any specificity such as id or class to `vaadin-context-menu-list-box`.